font/src/ft/mod.rs

```diff
@@ -265,13 +269,25 @@ impl FreeTypeRasterizer {
                 Some(FixedSize { pixelsize: pixelsize.next().expect("has 1+ pixelsize") })
             };
 
+            let pixelsize_fixup_factor = pattern.pixelsizefixupfactor().next().unwrap_or(0.);
```

Comment:
When is this going to be `None`, is this the case when the user has disabled bitmap scaling?

Do we really want to unwrap to `0` here? I feel like it should be an `Option` if it's potentially missing.

We later check for `pixelsize_fixup_factor == 0.` too, checking with `match` would be nicer.